AstroCal/control/control.py

```diff
@@ -206,3 +207,32 @@ def getMoonStatus():
 
     moon_percent_rounded = round(moon_percent)
     return result, "Illumination " + str(moon_percent_rounded) + "%"
+
+
+
+def getVariableDayLength(year, month, day, amountOfDays):
+    """Return 2d array with amount of hours and minutes per day from current day to specified amount of days
+    """
+    amountOfDayLight = []
+
+    for i in range(0,amountOfDays): 
+        daySunRise = getRiseSet(year, month, day + i, 'SUN', 'RISE')
+        daySunSet = getRiseSet(year, month, day + i, 'SUN', 'SET')
+        #convert to datetime object to perfrom difference
+        dayRiseTime = datetime(daySunRise[0], daySunRise[1], daySunRise[2], daySunRise[3], daySunRise[4], int(daySunRise[5]))
+        daySetTime = datetime(daySunSet[0], daySunSet[1], daySunSet[2], daySunSet[3], daySunSet[4], int(daySunSet[5]))
+        
+        #get difference between sunrise and and sunset 
+        timeDiff = abs(daySetTime-dayRiseTime)
+        #convert time difference into seconds
+        seconds = timeDiff.seconds
+        #extract total hours from seconds
+        dayLightHours = (seconds / 60) / 60
+        #extract remaining minutes
+        dayLightMinutes = dayLightHours % 1
```

Comment:
This variable is called minutes which implies the value is the number of minutes, but it's the decimal value of hours. Might be confusing to future users of the code base. Could we use a variable name like dayLightHoursRemainder?

Reply:
or it looks like there could be a more python-ic (is that a word?) way https://stackoverflow.com/a/775075/5708566